containers/ansible.cfg

```diff
@@ -0,0 +1,4 @@
+[defaults]
+vault_password_file = .vault_pass
+retry_files_enabled = False
+roles_path=roles:$PWD/../playbooks/roles:ansible-deployment/roles
```

Comment:
If this is supposed to refer to the main directory then it needs to drop the playbooks.